test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -711,8 +711,62 @@ namespace StackTest {
             $longPipeName = [string]::new("A", 200)
 
             "`$pid" | & $powershell -CustomPipeName $longPipeName -c -
-            # 64 is the ExitCode for BadCommandLineParameter
-            $LASTEXITCODE | Should -Be 64
+            $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
+        }
+    }
+
+    Context "ApartmentState WPF tests" -Tag Slow {
```

Comment:
@SteveL-MSFT Just FYI: When using more than 1 block (e.g. `Describe` and `Context`), then `Pester` v4 can only use tags on the outermost tag, meaning that the following does not execute any tests
```pwsh
Start-PSPester $pwd\test\powershell\Host\ConsoleHost.Tests.ps1 -Tag Slow
```
But since Pester v5 will start supporting this scenario, we can probably leave the tag as it is for the moment:
https://github.com/pester/Pester/issues/657